panel/src/panel/content.js

```diff
@@ -324,6 +328,23 @@ export default (panel) => {
 		 */
 		updateLazy(values = {}, env = {}) {
 			this.saveLazy(this.merge(values, env), env);
+		},
+
+		/**
+		 * Returns a specific version of the content
+		 * @param {String} versionId
+		 * @returns {Object|undefined}
+		 */
+		version(versionId) {
+			return this.versions()[versionId];
+		},
+
+		/**
+		 * Returns all versions of the content
+		 * @returns {Object}
+		 */
+		versions() {
```

Comment:
`Question:` Would this also work with a getter instead of a method?

Reply:
I looked again at all the env usage and I think I want to keep it. There is definitely some potential future benefit to it and we definitely need it for the requests. But that means that we hardly ever can "just" have a getter or we have to mix parts that honor the env and parts that don't. I haven't added env support to the versions method yet, but when we declare it as a method, we keep that option at least to also have an env argument for it later.

Reply:
That's what I created `panel.content.version("latest")` for

Reply:
I don't think we should have direct methods for versions. If we think ahead and there are more potential versions than just latest and changes, we can no longer follow that pattern.